unit_tests/rms_tests.cc

```diff
@@ -0,0 +1,81 @@
+/* daft engineers vocoder project
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+#include "../include/rms/rms.hh"
+#include <cmath>
+#include <gtest/gtest.h>
+#include <mutex>
+#include <thread>
+
+// This is a unit test, magic numbers are called test data here
+// NOLINTBEGIN(cppcoreguidelines-avoid-magic-numbers)
+TEST(RMSTests, buffer) {
```

Comment:
Can you copy in the NOLINTNEXTLINE that you use for the other test here too?

Reply:
done